envbuilder.go

```diff
@@ -656,6 +656,7 @@ func Run(ctx context.Context, options Options) error {
 				Insecure:      options.Insecure,
 				InsecurePull:  options.Insecure,
 				SkipTLSVerify: options.Insecure,
+				RegistryMirrors: strings.Split(os.Getenv("KANIKO_REGISTRY_MIRROR"), ";"),
```

Comment:
What do you think about splitting on ` ` instead? `;` seems a bit less standard to me.

It'd be great to add a comment linking to this PR and [these docs](https://github.com/GoogleContainerTools/kaniko?tab=readme-ov-file#flag---registry-mirror).

Reply:
I references the environment variable design in Kaniko [(link)](https://github.com/GoogleContainerTools/kaniko?tab=readme-ov-file#flag---registry-map), it uses `;` to separate multiple registry map. 
I think it might be better if we follows Kaniko's environment variable format.

Reply:
Agreed!